cli/tests/vanilla.test.ts

```diff
@@ -0,0 +1,1109 @@
+import {
+    checkVerifyingKeys,
+    deploy,
+    deployPoll,
+    deployVkRegistryContract, 
+    genProofs, 
+    mergeMessages, 
+    mergeSignups, 
+    proveOnChain, 
+    publish, 
+    setVerifyingKeys, 
+    signup, 
+    timeTravel,
+    verify
+} from "../ts/commands"
+import {
+    CheckVerifyingKeysArgs,
+    DeployPollArgs,
+    DeployVkRegistryArgs, 
+    GenProofsArgs, 
+    MergeMessagesArgs, 
+    MergeSignupsArgs, 
+    ProveOnChainArgs, 
+    PublishArgs, 
+    SetVerifyingKeysArgs, 
+    SignUpArgs, 
+    TimeTravelArgs,
+    VerifyArgs
+} from "../ts/utils"
+import { 
+    INT_STATE_TREE_DEPTH, 
+    MSG_BATCH_DEPTH, 
+    MSG_TREE_DEPTH, 
+    STATE_TREE_DEPTH, 
+    VOTE_OPTION_TREE_DEPTH 
+} from "./constants"
+import { homedir } from "os"
+import { cleanVanilla } from "./utils"
+
+describe("Vanilla tests", function() {
```

Comment:
Why are these "vanilla" tests? What does that signify?

Reply:
I think what originally was meant by vanilla is standard tests without the subsidy feature..

Reply:
what's a better name? will probably just change it to e2e?